students/Sean_Tasaki/Lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,102 @@
+"""
+Sean Tasaki
+5/18/2018
+Lesson05.mailroom_part_3
+"""
+
+donor_full_list = [['Bob Dylan', 120000.00, 3, 40000.00], ['Leonard Cohen', 600.00, 2, 300.00], ['Soren Kierkegaard', 1000.00, 2, 500.00], ['Italo Calvino', 20000.00, 4, 5000.00], ['Florence Feist', 300.00, 3, 100.00]]
+donor_name_list = []
+for x in donor_full_list:
+    donor_name_list.append(x[0])
+
+
+def main_menu():
+    main_menu_dict = {'1': thank_you, '2': create_report, '3': create_thank_you_letters, '4': quit}
+    main_prompt = "Enter 1-4 from the following options: (1) Send a Thank You to a Donor, (2) Create a Report, (3) Write a Thank You Letter to All Donors (4) Quit\n >> "  
+    main_menu_response(main_prompt, main_menu_dict)
+
+def main_menu_response(prompt, main_menu_dict):
+    try:
+        while True:
+            response = input(prompt)
+            main_menu_dict.get(response)()
+            if main_menu_dict[response]() == "exit menu":
+                exit()
+    except TypeError:
```

Comment:
you should have your try/except around the code that can throw that exception (line 23)